qsiprep/interfaces/dipy.py

```diff
@@ -645,6 +645,37 @@ class TensorReconstructionOutputSpec(DipyReconOutputSpec):
     cnr_image = File()
 
 
+class TensorReconstruction(DipyReconInterface):
```

Comment:
Should this really be `TensorReconstruction`? I might be misunderstanding something, but it seems to be over-ridden on line 679?